File: PC/marina/Marina/Classes/Entity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Marina.Classes
{
    public class Entity
    {
        private string value;

        // type of the entity
        private string type;

        public Entity(string v, string t)
        {
            value = v;
            type = t;
        }

        public string getValue(){
            return value;
        }
    

        public string getType()
        {
            return type;
        }
    }
}

File: android/Marina/app/src/main/java/com/example/user/marina/Entity.java
package com.example.user.marina;

public class Entity {

    String entity;
    String type;

    public Entity(String entity, String type)
    {
        this.entity = entity;
        this.type = type;
    }

    public String getType()
    {
        return type;
    }

    public String getEntity()
    {
        return entity;
    }
}